packages/surveycto/package.json

```diff
@@ -22,6 +22,7 @@
   ],
   "dependencies": {
     "@openfn/language-common": "workspace:*",
+    "date-fns": "^3.6.0",
```

Comment:
Wow, really?

We need a changeset attached to this because it probably means the adaptor has broken in production for any functions that use date-fns

Reply:
I have removed it. I was having trouble with the tests for `surveyCTO` and I had to upgrade babel/core in `/tools/simple-ast` which made it work